<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Gary Twinn gary@twinn.net</author>
 <class>dialogXYSetup</class>
 <widget class="QDialog" name="dialogXYSetup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calibrate X-Y Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/main/iconGTRun.svg</normaloff>:/main/iconGTRun.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lineYPosition">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineXPosition">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnUp">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move Up</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/xy setup/arrorwe up.png</normaloff>
     <normalon>:/xy setup/arrorw up.png</normalon>:/xy setup/arrorwe up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnLeft">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move Left</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/xy setup/arrorwe left.png</normaloff>
     <normalon>:/xy setup/arrorw left.png</normalon>:/xy setup/arrorwe left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnRight">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move Right</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/xy setup/arrorwe right.png</normaloff>
     <normalon>:/xy setup/arrorw right.png</normalon>:/xy setup/arrorwe right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDown">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move Down</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/xy setup/arrorwe down.png</normaloff>
     <normalon>:/xy setup/arrorw down.png</normalon>:/xy setup/arrorwe down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboLocation1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGoto">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Goto the location specified</string>
   </property>
   <property name="text">
    <string>Goto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>170</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="Saves the current X and Y position to the location shown" extracomment="Saves the current X and Y position to the location shown">Saves the current X and Y position to the location shown</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string>Saves the current X and Y position to the location shown</string>
   </property>
   <property name="whatsThis">
    <string>Saves the current X and Y position to the location shown</string>
   </property>
   <property name="text">
    <string>Save Current</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Stop the stage moving</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Warning&lt;/span&gt;: Please ensure that the X-Y table has finished moving before pressing &lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Save Current&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Control</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGotoNext">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Goto Next</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
